openfisca_france/parameters/prelevements_sociaux/professions_liberales/auto_entrepreneur/cotisations_prestations/service.yaml

```
@@ -52,6 +58,21 @@ metadata:
     2022-10-01:
       title: Décret n° 2022-1529 du 7 décembre 2022
       href: https://www.legifrance.gouv.fr/jorf/article_jo/JORFARTI000046710847
+    2024-07-01:
+      - title: Décret n° 2024-484 du 30 mai 2024 - art. 1
+        href: https://www.legifrance.gouv.fr/loda/article_lc/LEGIARTI000049622560/2024-06-01
+      - title: Article D613-4 du CSS
+        href: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000043656794/2024-07-01/
+    2025-01-01:
+      - title: Décret n° 2024-484 du 30 mai 2024 - art. 1
+        href: https://www.legifrance.gouv.fr/loda/article_lc/LEGIARTI000049622560/2024-06-01
+      - title: Article D613-4 du CSS
+        href: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000043656794/2025-01-01/
+    2026-01-01:
+      - title: Décret n° 2024-484 du 30 mai 2024 - art. 1
+        href: https://www.legifrance.gouv.fr/loda/article_lc/LEGIARTI000049622560/2024-06-01
+      - title: Article D613-4 du CSS
+        href: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000043656794/2026-01-01/
```

```suggestion
      - title: Article D613-4 du CSS
        href: https://www.legifrance.gouv.fr/jorf/article_jo/JORFARTI000049620959
```
Idem. Par ailleurs, je crois que la règle est de ne pas mettre de valeurs futures dans les paramètres ?